user_sync/config.py

```diff
@@ -497,7 +497,19 @@ def get_rule_options(self):
 
         # get the limits
         limits_config = self.main_config.get_dict_config('limits')
-        options['max_adobe_only_users'] = limits_config.get_int('max_adobe_only_users')
+        max_missing = limits_config.get_value('max_adobe_only_users',(int, str),False)
+        if isinstance(max_missing, str):
+            percent_pattern = re.compile("(\d*(\.\d+)?%)")
+            if percent_pattern.match(max_missing):
+                max_missing_percent = float(max_missing.strip('%'))
+                if max_missing_percent >= 0.0 and max_missing_percent <= 100.0:
+                    options['max_adobe_only_users'] = max_missing
+                else:
+                    raise AssertionException("max_adobe_only_users value must be less or equal than 100%")
+            else:
+                raise AssertionException("Not a valid value for max_adobe_only_users. Must be a percentage or fix integer")
```

Comment:
We don't need the word "fix" here - just change it to read "a percentage or an integer".